backend/dal/Services/Concrete/ParcelService.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Collections.Generic;
+using Pims.Dal.Entities;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using System.Security.Claims;
+
+namespace Pims.Dal.Services.Concrete
+{
+    /**
+     * EF Core implementation of parcel service
+     */
+    public class ParcelService : IParcelService
+    {
+        private readonly PIMSContext _dbContext;
+        private readonly ClaimsPrincipal _user;
+        public ParcelService(PIMSContext dbContext, ClaimsPrincipal user)
+        {
+            _dbContext = dbContext;
+            _user = user;
+        }
+
+        public IEnumerable<Parcel> GetParcels(double? neLat = null, double? neLong = null, double? swLat = null, double? swLong = null)
```

Comment:
Nullable arguments make me nervous.  Does this function work if any are null?  I mean in a realistic way.